src/org/sosy_lab/common/configuration/ConfigurationBuilder.java

```diff
@@ -250,35 +253,72 @@ public ConfigurationBuilder loadFromFile(Path file)
    * @throws IllegalArgumentException If the resource cannot be found or read, or contains invalid
    *     syntax or #include directives.
    */
+  @SuppressWarnings("try")
```

Comment:
Why is this necessary?

Reply:
Because the FileSystem in the try-with-ressources-block is never used locally.